app/views/moderator_applications/index.html.slim

```diff
@@ -0,0 +1,14 @@
+.container-fluid
+  .row
+    .col-lg-2
+
+    main.col-lg-8
+      h1 Moderator Applications
+      ul
+        - @applications.each do |application|
+          li
+            = link_to(application.user.username, moderator_application_path(application.id))
```

Comment:
Inconsistency: this should be just `moderator_application_path(application)`, sans the `.id`.